filtered-resource-service/README.md

```diff
@@ -18,70 +18,92 @@ limitations under the License.
 
 # Filtered Resource Service
 
-The Filtered-Resource Service sits at the end of the stream pipeline and has the task of returning resources (and their connection details) back to the client. Given a token `$[token]` from the Palisade Service, it is accessed by a Websocket request
-to `ws://filtered-resource-service/resource/${token}`. The service will return each resource discovered by the Palisade system one-by-one to the client as requested, auditing that the client is now aware of these resources.
+![Filtered-Resource Service High-Level Diagram](doc/Filtered-Resource-Service-HL.png)
 
-## High-Level Architecture
+The Filtered-Resource Service sits at the end of the stream pipeline and has the task of returning resources (and their connection details) back to the client. 
```

Comment:
No mention of Data Service, is that intentional?

Reply:
Yep, there's no real need to mention it here, if all you needed from Palisade was an overblown `ls` system.
I will mention that the expectation is that these responses would likely be requested to download from the data-service nonetheless.

Reply:
I've made a preference for 'Some-Thing Service'